README.md

```
@@ -1,8 +1,8 @@
 # Golang API client for Uploadcare
 
 ![license](https://img.shields.io/badge/license-MIT-brightgreen.svg)
-[![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)](http://godoc.org/github.com/uploadcare/uploadcare-go/ucare)
-![](https://github.com/uploadcare/uploadcare-go/workflows/test/badge.svg)
+[![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)](http://pkg.go.dev/github.com/uploadcare/uploadcare-go/v2/ucare)
+![](https://github.com/uploadcare/uploadcare-go/v2/workflows/test/badge.svg)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)